### PR TITLE
streamline the constraints reference

### PR DESCRIPTION
**Pull request by OskarStark**
I felt the constraints need some love 💟

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Thanks a lot Oskar! This probably took you some time, so we appreciate that!

File: reference/constraints/Bic.rst [hunk 1]
```diff
@@ -73,8 +73,6 @@ will contain a Business Identifier Code (BIC).
 
         class Transaction
         {
-            protected $businessIdentifierCode;
```
Comment (xabbuh) [inline reference/constraints/Bic.rst]: @OskarStark Wouldn't it be less confusing to always add the property instead?

Comment (OskarStark) [reply reference/constraints/Bic.rst]: Ofc we can, but most of them does *not* have them.

You get my 👍

Comment (OskarStark) [reply reference/constraints/Bic.rst]: Ofc we can, but most of them does *not* have them.

You get my 👍

**Pull request merged**